### PR TITLE
Adds scrollable parameter to the modal tag.

### PR DESCRIPTION
**Pull request by bwiedmann**
Fixes: SIRI-266

### CODE REVIEW

File: src/main/resources/default/taglib/w/modal.html.pasta [hunk 1]
```diff
@@ -1,11 +1,12 @@
 <i:arg name="name" type="String" />
 <i:arg name="titleKey" type="String" default="" />
 <i:arg name="title" type="String" default="@i18n(titleKey)" />
+<i:arg name="scrollable" type="boolean" default="false" />
 
 <i:pragma name="description" value="Renders a body for a modal dialog within a Wondergem template" />
 
 <div id="@name" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="@(name + ' Title')">
-    <div class="modal-dialog" role="document">
+    <div class="modal-dialog @if(scrollable){modal-dialog-scrollable}" role="document">
```
Comment (andyHa) [inline src/main/resources/default/taglib/w/modal.html.pasta:9]: @modal-dialog-scrollable="scrollable"

**Review by andyHa — CHANGES_REQUESTED** [review]
..

File: src/main/resources/default/taglib/w/modal.html.pasta (hunk 1 above)
Comment (bwiedmann) [reply src/main/resources/default/taglib/w/modal.html.pasta:9]: Resulted in "Unknown variable modal"
`       <div class="modal-dialog @modal-dialog-scrollable='scrollable'" role="document">
`

**Review by idlira — APPROVED** [review]

**Review by andyHa — APPROVED** [review]

**Comment by bwiedmann** [comment]
As it's Bootstrap 4 specific code it will never work. The scrollable logic will be implemented in css in biz.

**Pull request closed without merging**